### PR TITLE
Jack commit first in training

### PR DESCRIPTION
**Pull request by JackZhong4Openlane**

### CODE REVIEW

File: _posts/0000-01-11-JackZhong4Openlane.md [hunk 1]
```diff
@@ -5,5 +5,7 @@ title: "Welcome to our slide deck!"
```
Comment (lpofxenon) [inline _posts/0000-01-11-JackZhong4Openlane.md]: Hey, maybe you should change this title.

File: _posts/0000-01-11-JackZhong4Openlane.md [hunk 2]
```diff
@@ -5,5 +5,7 @@ title: "Welcome to our slide deck!"
 
 Caption here
```
Comment (ol-vrobison) [inline _posts/0000-01-11-JackZhong4Openlane.md:6]: You should have put the caption here instead

**Review by JackZhong4Openlane — COMMENTED** [review]
looks good

File: _posts/0000-01-11-JackZhong4Openlane.md (hunk 2 above)
Comment (lpofxenon) [inline _posts/0000-01-11-JackZhong4Openlane.md]: Hey, maybe you need to change the caption here.

Comment (judyzhu-adesa) [inline _posts/0000-01-11-JackZhong4Openlane.md:6]: Hey, maybe you need to change the caption here.

**Review by judyzhu-adesa — CHANGES_REQUESTED** [review]
Please change the caption on the proper line.

**Review by ol-vrobison — DISMISSED** [review]
@JackZhong4Openlane  caption not really changed where the "Caption Here text was"

**Review by judyzhu-adesa — APPROVED** [review]

File: _posts/0000-01-11-JackZhong4Openlane.md (hunk 2 above)
Comment (JackZhong4Openlane) [reply _posts/0000-01-11-JackZhong4Openlane.md:6]: Victor , please review and approve my request.

**Review by judyzhu-adesa — APPROVED** [review]

**Pull request merged**